### ISSUE TITLE
Add option to skip or configure Prettier formatting

### ISSUE THREAD

**Issue by bengry** [jarlef/vite-plugin-preload#3]
Prettier formatting is nice, but it causes some unexpected consequences if you rely on modifying the HTML after the fact (e.g. if you have a CI pipeline that does a find & replace on stuff).

I suggest either removing Prettier formatting completely or allowing the consumer to specify options. e.g.
```ts
// vite.config.ts
import { defineConfig } from "vite";
import react from "@vitejs/plugin-react";
import preload from "vite-plugin-preload";

export default defineConfig({
  plugins: [
    react(),
    preload({ format: false }), // to disable prettier formatting completely
    preload({ // or specify options to pass to `prettier.format`
      format: {
        /* Omit<prettier.Options, 'parser'> */
        embeddedLanguageFormatting: "off",
      },
    }),
  ],
});
```

**Comment by bengry** [comment]
I ended up forking the package and publishing it under [vite-plugin-preload-2](https://www.npmjs.com/package/vite-plugin-preload-2) if anyone is interested in this fix.

**Comment by jarlef** [comment]
Sorry about the late reply. I will take a look at it asap

**Comment by bengry** [comment]
> Sorry about the late reply. I will take a look at it asap

Thanks! Also notice that my fork has #5's fix merged.

**Comment by jarlef** [comment]
Fix has been published in version 0.3.0
